src/_variables.scss

```diff
@@ -0,0 +1,56 @@
+// Core variables
+// Global
+$html-font-size: 10px;
+$html-line-height: 1.428571429;
+$body-bg: #fff;
+$body-font-color: #333;
+$body-font-size: 1.4rem;
+$body-font-family: "Helvetica Neue", "PingFang SC", "Hiragino Sans GB", "Microsoft YaHei", "Hiragino Kaku Gothic Pro", Meiryo, "Malgun Gothic", sans-serif;
+
+// Sizes
+$layout-gutter: 1rem;
+
+// Colors
+$core-color: #5764c6;
```

Comment:
Add the `!default` keyword at the end of variables to allow us to override them.

```sass
$core-color: red;
@import "spectre.css/src/spectre.scss";
```

[!default keyword on SASS doc](http://sass-lang.com/documentation/file.SASS_REFERENCE.html#variable_defaults_)